src/plugins/intel_cpu/src/infer_request.cpp

```diff
@@ -336,6 +337,10 @@ ov::intel_cpu::MKLDNNInferRequestBase::normToInputSupportedPrec(const std::pair<
     return inPrec;
 }
 
+void ov::intel_cpu::MKLDNNInferRequestBase::SetBlobsImpl(const std::string& name, const InferenceEngine::BatchedBlob::Ptr& batched_blob) {
```

Comment:
since this feature only for API 2.0, let's override all methods only for `MKLDNNInferRequest`

Reply:
Agree, done